packages/circuits/package.json

```diff
@@ -4,34 +4,30 @@
     "main": "dist/src/index.js",
     "types": "dist/src/index.d.ts",
     "description": "ZK proofs used for the unirep attesters",
-    "author": "Unirep social TW",
-    "homepage": "https://github.com/social-tw/social-tw-website",
+    "author": "Unirep Team <team@unirep.io>",
+    "homepage": "https://github.com/Unirep/create-unirep-app#readme",
     "license": "MIT",
     "repository": {
         "type": "git",
-        "url": "https://github.com/social-tw/social-tw-website.git"
+        "url": "git+https://github.com/Unirep/create-unirep-app.git"
     },
     "scripts": {
         "build": "tsc && yarn copyCircom",
-        "buildOverwrite": "find ./zksnarkBuild -type f -not -name '*.ptau' -delete && yarn buildsnark",
         "buildsnark": "ts-node ./scripts/buildSnarks.ts && tsc && yarn copyCircom",
         "test": "mocha -r ts-node/register test/*.test.ts --exit",
         "copyCircom": "sh scripts/copyCircom.sh"
     },
     "bugs": {
-        "url": "https://github.com/social-tw/social-tw-website/issues"
+        "url": "https://github.com/Unirep/create-unirep-app/issues"
```

Comment:
should modify to previous description

Reply:
Thank you! I believe it's all fixed, could you take a look again